### DISCUSSION TITLE
New Component: Skeleton

### DISCUSSION THREAD

**Discussion by Mil4n0r** [dxc-technology/halstack-react#1772] [Ideas]
**Describe the solution you'd like**
I propose the addition of a new `Skeleton` component to our design system, aimed at addressing various use cases. The primary objective is to enhance the user experience by significantly reducing the perceived loading time, thereby minimizing potential frustration in specific scenarios.

This component usually looks similar to this:

![image](https://github.com/dxc-technology/halstack-react/assets/33158831/648c6746-2df6-4ff8-b13f-3d1374c9a07b)

**Describe alternatives you've considered**
While we currently use the Spinner component, the Skeleton offers a superior approach in modern UIs. Its ability to provide a preview of the content structure through visual cues not only reduces perceived loading time but also contributes to a more professional and polished style.

**Additional context**
The proposed Skeleton component is designed to be straightforward, featuring customizable attributes such as `height` and `width`, with an optional `animation`. Looking ahead, we can collaborate with the design team to explore additional configurations like `shape`, `color` and `animation-speed` to further enhance the User Experience.

**Comment by GomezIvann** [comment]
Enrique this is a great proposal! But I think it fits better inside the [Discussions](https://github.com/dxc-technology/halstack-react/discussions) page since it requires debating its use case, priority, viability and other common matters from these kind of situations.

**Comment by Mil4n0r** [comment]
I moved this to [Discussions](https://github.com/dxc-technology/halstack-react/discussions)
